charts/illumidesk/values.yaml

```diff
@@ -157,10 +157,17 @@ jupyterhub:
           configMapKeyRef:
             name: hub-illumidesk-cm           # The ConfigMap this value comes from.
             key: POSTGRES_JUPYTERHUB_DB
+      ACTIVE_USER_LIMIT:
+        valueFrom:
+          configMapKeyRef:
+            name: hub-illumidesk-cm           # The ConfigMap this value comes from.
+            key: ACTIVE_USER_LIMIT
       LTI13_PRIVATE_KEY: "/srv/jupyterhub/rsa_private.key"
       # Mount Root
       ILLUMIDESK_MNT_ROOT: "/illumidesk-courses"
     extraConfig:
+      serverLimitConfig:
+        c.JupyterHub.active_server_limit = int(os.environ.get('ACTIVE_USER_LIMIT'))
```

Comment:
Is there any particular reason we are adding the config in the `extraConfig` key instead of using the standard `hub.activeServerLimit` key?